build.py

```diff
@@ -80,10 +80,13 @@ def build():
                              utility.ROOT_DIR,
                              continue_on_error=False)
 
-        # checks if long description will render correctly
-        utility.exec_command('twine check {}'
-                             .format(os.path.join(utility.MSSQLCLI_DIST_DIRECTORY, '*')),
-                             utility.ROOT_DIR, continue_on_error=False)
+        try:
+            # checks if long description will render correctly--does not work on some systems
+            utility.exec_command('twine check {}'
+                                .format(os.path.join(utility.MSSQLCLI_DIST_DIRECTORY, '*')),
+                                utility.ROOT_DIR, continue_on_error=False)
+        except IOError:
+            print("Unable to run 'twine check'.")
```

Comment:
Let's print the exception error message and stack trace so that someone could troubleshoot